symforce/codegen/backends/cpp/cpp_config.py

```diff
@@ -61,13 +70,12 @@ def templates_to_render(self, generated_file_name: str) -> T.List[T.Tuple[str, s
         return templates
 
     def printer(self) -> CodePrinter:
-        # NOTE(hayk): Is there any benefit to this being lazy?
-        from symforce.codegen.backends.cpp import cpp_code_printer
```

Comment:
I'd probably either leave this import here, or make it eager for all the printers (i.e. also python_config.py)

Reply:
Yeah, I saw Hayk's comment and I couldn't think of any benefit of it's laziness but you obviously have some more insight here. Seems cleanest to me just make it a standard import everywhere then

Reply:
Yeah I don't think there's any reason for it to be lazy, go for it